controllers/posts.js

```diff
@@ -6,8 +6,9 @@ const PostsController = {
       if (err) {
         throw err;
       }
-      res.render("posts/index", { posts: posts.reverse() });
-    });
+      res.render("posts/index", { posts: posts });
+      console.log(posts)
+    }).sort( { createdAt: 'asc' } ).exec();
   },
```

Comment:
Currently the latest post goes to the bottom of the posts page, which needs to change so that the latest post goes to the top of the page. i.e(change 'asc' in line 11 to 'desc'

Reply:
That's been amended and I've stripped out a log statement that didn't need to be there. Thanks Sanam for spying the quick fix!